oauth2/fosite_store_test.go

```diff
@@ -141,17 +132,7 @@ func TestManagers(t *testing.T) {
 	}
 
 	for k, store := range fositeStores {
-		if k != "memory" {
```

Comment:
Moved these to the exported package to make it easier to track changes to tests.